modules/solid_mechanics/include/bcs/ExplicitFunctionDirichletBC.h

```diff
@@ -18,12 +18,12 @@ class Function;
  * function at the boundary.
  * For use with direct central difference time integrator
  */
-class DirectFunctionDirichletBC : public DirectDirichletBCBase
+class ExplicitFunctionDirichletBC : public ExplicitDirichletBCBase
```

Comment:
This object currently produces an error that could be improved (I think it mentions that no mass matrix inverse could be found). Maybe pointing the user to an explicit TI, like `ExplicitMixedOrder` would be more helpful.

Reply:
Yup will point that error to `ExplicitMixedOrder`.